Cucumber/features/project1/pages/find_patient_page.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'top_part.rb'
+class FindPatientPage
+  attr_accessor :find_patient_btn,:table_column_content
+
+  include TopPart
+
+  def initialize(browser)
+    @driver = browser
+    @find_patient_search_field = @driver.find_element(id: 'patient-search')
+  end
```

Comment:
I thought BasicPage is for inheriting it in all others? If the mains idea of that page is so, You can just inherit it and in initialize call super